### PR TITLE
Refactor the get users endpoint to also fetch respective projects and apps 

### PR DESCRIPTION
**Pull request by LanternNassi**
# Description

This is to add extra information to an account's profile when being fetched through adding their respective projects and apps

## Type of change

Please delete options that are not relevant.
- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

## Trello Ticket ID

https://trello.com/c/wCa2h1sh/1557-user-profile-endpoint-add-missing-data

## How Can This Be Tested?
Supplying the entire_profile parameter to the Get users endpoint 

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by inno-asiimwe** [comment]
This may be a lot of data to return within a single endpoint. (When you think about it at scale)

**Comment by LanternNassi** [comment]
> This may be a lot of data to return within a single endpoint. (When you think about it at scale)

@inno-asiimwe . I think pagination will help us overcome that .

**Review by Mubangizi — APPROVED** [review]

File: app/controllers/users.py [hunk 1]
```diff
@@ -138,12 +142,41 @@ def get(self):
                 'per_page': paginated.per_page,
                 'next': paginated.next_num,
                 'prev': paginated.prev_num
-            } 
-
+            }
+        
+
```
Comment (jnsereko) [inline app/controllers/users.py:147]: extra space might not be too helpful

File: app/controllers/users.py [hunk 2]
```diff
@@ -138,12 +142,41 @@ def get(self):
...
+                    Project_schema = ProjectSchema()
+                    project_data , errors = Project_schema.dumps(project)
+                    project_info = {**json.loads(project_data)}
+
+                    App_schema = AppSchema(many=True)
+                    App_data , errors = App_schema.dumps(project.apps)
+                    project_info['apps'] = App_data
+                    user_profile['Projects'].append(project_info)
+
+
```
Comment (jnsereko) [inline app/controllers/users.py:173]: same here

**Review by jnsereko — APPROVED** [review]
looks fine 
Thanks @LanternNassi

**Review by MutegekiHenry — APPROVED** [review]
LGTM

**Pull request merged**